lib/hyperion/aux/logger.rb

```diff
@@ -23,7 +27,16 @@ def log_stub(rule)
     private
 
     def log_headers(headers, logger)
-      headers.each_pair { |k, v| logger.info "    #{k}: #{v}" unless k == 'Expect' }
+      h = headers.keep_if { |k| k != 'Expect' }
+      logger.info(log_message(h))
+    end
+
+    def log_message(obj)
```

Comment:
We are expecting the error response to be "Hyperion Compliant" which is json `ClientErrorResponse`